src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -10096,6 +10096,7 @@ interface IStyleSettings {
  */
 interface GameInfo {
   version: string;
+  versionNumber: number;
```

Comment:
```suggestion
  /** Internal version number that increments during releases. E.g.: 43 */
  versionNumber: number;
```
Please correct the examples if they are not valid current/past examples.